### ISSUE TITLE
Error Code: -32603 

### ISSUE THREAD

**Issue by goldprince48** [zndn1997/vscode-pawn#4]
Every time I try to put round brackets after a function name, This error message pops up in output tab, and it's really annoying.

Message: 
`[Error - 1:11:48 am] Request textDocument/signatureHelp failed.
  Message: Request textDocument/signatureHelp failed with message: Cannot read property 'map' of undefined
  Code: -32603 `

This always pops when you try to put round brackets after a function name, For example: if I try to put `()` after writing `strcat` this error message will pop up. This mostly happens while working on ".inc" file.

**Comment by qoor** [comment]
Yes, this is an issue I already know. Maybe this is a vscode extension part script error.

And this is WIP extension. But I have a plan to cross platform native PAWN LSP Server write in C++.
